diff --git a/tec_uname b/tec_uname
index 2d2b6a9..0b6a624 100755
--- a/tec_uname
+++ b/tec_uname
@@ -143,16 +143,16 @@ ComputeTecUname()
 ComputeSystemPaths()
 {
   if [ $TEC_SYSARCH == x64 ]; then
-    if [ -d /usr/lib64 ]; then
-      TEC_SYSTEM_LIB=/usr/lib64
+    if [ -d /app/lib64 ]; then
+      TEC_SYSTEM_LIB=/app/lib64
     else
-      TEC_SYSTEM_LIB=/usr/lib
+      TEC_SYSTEM_LIB=/app/lib
     fi
   else
-    TEC_SYSTEM_LIB=/usr/lib
+    TEC_SYSTEM_LIB=/app/lib
   fi
 
-  TEC_SYSTEM_INC=/usr/include
+  TEC_SYSTEM_INC=/app/include
   
   if [ $TEC_SYSNAME == Haiku ]; then
     TEC_SYSTEM_LIB=`finddir B_SYSTEM_LIB_DIRECTORY`
